python_modules/dagster/dagster/_core/definitions/op_invocation.py

```diff
@@ -59,31 +121,24 @@ def op_invocation_result(
     context = context or build_op_context()
 
     resource_arg_mapping = {arg.name: arg.name for arg in compute_fn.get_resource_args()}
-    resource_args_from_kwargs = {}
-    for resource_arg in resource_arg_mapping:
-        if resource_arg in kwargs:
-            resource_args_from_kwargs[resource_arg] = kwargs[resource_arg]
-            del kwargs[resource_arg]
 
-    resources_provided_in_multiple_places = (resource_args_from_kwargs) and (context.resource_keys)
+    # Extract any resource inputs from the args and kwargs
+    extracted = _extract_resource_args(compute_fn, args, kwargs, resource_arg_mapping)
+
+    adjusted_args = extracted.args
+    adjusted_kwargs = extracted.kwargs
```

Comment:
should be say `input_args` and `input_kwargs`? I think that is what these actually are.